### PR TITLE
MTM-60761 - RN: Latest measurement values don't work for "*"

### PR DESCRIPTION
**Pull request by mbrz-c8y**
Ticket: [MTM-60761](https://cumulocity.atlassian.net/browse/MTM-60761)

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10-20-585-0-latest-measurements-wildcard.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+date: 2024-10-07
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-585-0-latest-measurements-wildcard.md]:
```suggestion
date:
```

Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10-20-585-0-latest-measurements-wildcard.md]: The date will be synchronized with the deployment of the version on Zone 1.

File: content/change-logs/platform-services/cumulocity-10-20-585-0-latest-measurements-wildcard.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-60761
+version: 10.20.585.0
+---
+
+Fixed tenant option used to enable [latest measurement values](https://cumulocity.com/docs/standard-tenant/managing-data/#latest-value) not working with the wildcard (`*`) key. After the fix the wildcard character can be used to enable latest measurements for all series:
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-585-0-latest-measurements-wildcard.md]:
```suggestion
Previously, the tenant option which is used to enable [latest measurement values](https://cumulocity.com/docs/standard-tenant/managing-data/#latest-value) was not not working with the wildcard (`*`) character. With this change, the wildcard character can be used to enable latest measurements for all series:
```

**Review by karol-c8y — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-585-0-latest-measurements-wildcard.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+date:
+title: Fixed latest measurement values tenant option with wildcard key
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-585-0-latest-measurements-wildcard.md]:
```suggestion
title: Latest measurement values tenant option now works with wildcard character
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**